bob-apps/bin/bobp.rs

```diff
@@ -438,46 +491,65 @@ fn print_periodic_stat(
         DiffContainer::new(Box::new(|| stat.get_time_ns_st.load(Ordering::Relaxed)));
     let mut get_count_st =
         DiffContainer::new(Box::new(|| stat.get_count_st.load(Ordering::Relaxed)));
+    let mut exist_time_st =
+        DiffContainer::new(Box::new(|| stat.exist_time_ns_st.load(Ordering::Relaxed)));
+    let mut exist_count_st =
+        DiffContainer::new(Box::new(|| stat.exist_count_st.load(Ordering::Relaxed)));
     let mut get_size = DiffContainer::new(Box::new(|| stat.get_size_bytes.load(Ordering::Relaxed)));
+    let mut exist_size = DiffContainer::new(Box::new(|| stat.exist_size_bytes.load(Ordering::Relaxed)));
     let mut put_speed_values = vec![];
     let mut get_speed_values = vec![];
-    let k = request_bytes as f64 / period_ms as f64 * 1000.0 / 1024.0;
+    let mut exist_speed_values = vec![];
+    let sec = period_ms as f64 / 1000.;
+    let k = request_bytes as f64 / 1024.0;
     let start = Instant::now();
     while !stop_token.load(Ordering::Relaxed) {
         thread::sleep(pause);
         let put_count_spd = put_count.get_diff() * 1000 / period_ms;
         let get_count_spd = get_count.get_diff() * 1000 / period_ms;
+        let exist_count_spd = exist_count.get_diff() * 1000 / period_ms;
         let cur_st_put_time = put_time_st.get_diff() as f64;
         let cur_st_put_count = put_count_st.get_diff() as f64;
         let cur_st_get_time = get_time_st.get_diff() as f64;
         let cur_st_get_count = get_count_st.get_diff() as f64;
+        let cur_st_exist_time = exist_time_st.get_diff() as f64;
+        let cur_st_exist_count = exist_count_st.get_diff() as f64;
 
         let put_error = stat.put_error_count.load(Ordering::Relaxed);
         let get_error = stat.get_error_count.load(Ordering::Relaxed);
+        let exist_error = stat.get_error_count.load(Ordering::Relaxed);
         let put_spd = put_count_spd as f64 * k;
-        let get_spd = get_size.get_diff() as f64 / 1024.0;
+        let get_spd = get_size.get_diff() as f64 / 1024. / sec;
+        let exist_spd = exist_size.get_diff() as f64 / 1024.0 / sec;
         if put_spd > 0.0 {
             put_speed_values.push(put_spd);
         }
         if get_spd > 0.0 {
             get_speed_values.push(get_spd);
         }
+        if exist_spd > 0.0 {
+            exist_speed_values.push(exist_spd);
+        }
         println!(
-            "put: {:>6} rps  | get {:>6} rps   | put err: {:5}     | get err: {:5}\r\n\
-            put: {:>6.2} kb/s | get: {:>6.2} kb/s | put lat: {:>6.2} ms | get lat: {:>6.2} ms",
+            "put: {:>6} rps  | get {:>6} rps   | exist {:>6} rps   | put err: {:5}     | get err: {:5}     | exist err: {:5}\r\n\
+            put: {:>6.2} kb/s | get: {:>6.2} kb/s | exist: {:>6.2} kb/s | put lat: {:>6.2} ms | get lat: {:>6.2} ms | exist lat: {:>6.2} ms",
```

Comment:
The line becomes too long. I suggest to print information only about operations that was actually executed